cif_pow.dic

```diff
@@ -6310,6 +6334,27 @@ save_pd_diffractogram.diffrn_id
 
 save_
 
+save_pd_diffractogram.diffrn_scan_id
+
+    _definition.id                '_pd_diffractogram.diffrn_scan_id'
```

Comment:
I suggest we leave off the `diffrn` part from the data name. It doesn't contribute anything.

Reply:
Concur. Changed.